pkg/util/date.go

```diff
@@ -724,3 +724,14 @@ func ConvertToPST(t time.Time) (*time.Time, error) {
 
 	return &pt, nil
 }
+
+// GetLocDateFromState gets the time based on the given state's timezone
+func GetLocDateFromState(state string) (*time.Time, error) {
```

Comment:
Can we also update the name of the function name as we are returning time and not date.